site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -82,7 +82,7 @@ class OAuth2Controller(
         withHeaders(HeaderNames.AUTHORIZATION → s"token $authToken").
         get().map { response ⇒
           val login = (response.json \ "login").as[String]
```

Comment:
There are several examples in the play docs with example on how to properly write a custom writer or reader